components/server/src/authorization/authorizer.ts

```diff
@@ -38,6 +38,22 @@ export class Authorizer {
         return this.authorizer.check(req, { orgID: orgId });
     }
 
+    async hasPermissionOnProject(userId: string, permission: ProjectPermission, project: Project): Promise<boolean> {
+        const req = v1.CheckPermissionRequest.create({
+            subject: subject("user", userId),
+            permission,
+            resource: objectRef("project", project.id),
+            consistency: v1.Consistency.create({
+                requirement: {
+                    oneofKind: "fullyConsistent",
+                    fullyConsistent: true,
+                },
```

Comment:
Extract this to a constant and re-use between `hasPermissionOnProject` and `hasPermissionOnOrg`?